src/forms/labels/labeled-field.js

```diff
@@ -40,6 +42,16 @@ import InputLabel from './input-label'
  *     </LabeledField>
  *   )
  * }
+ * 
+ * // A custom label to pass in as a label component (re-using `<InputLabel />`)
+ * 
+ * const handleClick = () => TermsModal.show()
+ * function CustomLabelComponent (props) {
```

Comment:
You should include an example of passing this to `LabeledPhoneInput`. Also, defining handleClick outside the component it a little unorthodox- perhaps pass it as another prop to `LabeledPhoneInput`?